packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -93,10 +90,12 @@ mi-dropdown-item {
         text-overflow: ellipsis;
 
         &--from-inner-html {
-            display: flex;
-            flex-direction: row;
+            display: grid;
             align-items: center;
+            grid-auto-flow: column;
+            grid-template-columns: variables.$spacing-large 1fr;
             gap: variables.$spacing-x-small;
+            justify-items: center;
```

Comment:
Could we use `place-items`?

Reply:
Refactored the code to use `place-items`